YACS-RCOS/yacs.n
# Adding summer 2021 files

**Issue**

Closes #400 

**Test Procedure**

Show procedure to test functionality with a clear procedure

*Example:*
> 1. Submit the summer2021 csv on the admin panel
> 2. Check that classes loaded in
